include/print_log.h

```diff
@@ -0,0 +1,110 @@
+/***************************************************************************//**
+ *   @file   print_log.h
```

Comment:
why not debug.h ?

Reply:
The file contains multiple log levels. Debug is only one of them.